code-challenges.test.js

```diff
@@ -90,4 +90,4 @@ const addUpVotes = (votes) => {
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code: I found the final result the total number of votes by subtracting downvotes from upvotes, then returned the total number of votes.
+// Pseudo code: I found the final result the total number of votes by subtracting downvotes from upvotes, then returned the total number of up votes and down votes.
```

Comment:
Looks great. This totally works. As a refactor you could consider reducing the number of lines by doing `return totalNumberOfVotes = votes.upVotes - votes`